### DISCUSSION TITLE
slider with html button for smartphone

### DISCUSSION THREAD

**Discussion by Nega100** [jspsych/jsPsych#3255] [Q&A]
Hi All!!
Is there any simple method to combine html slider and html button, so that the slider could run in smartphone (touchscreen) device. 
Many thanks.

**Comment by Shaobin-Jiang** [comment]
If you mean a slider with a confirmation button, you can just use the html-slider-response plugin. Other wise you could just use the html-button-response and set the `stimulus` to a slider.

**Reply by Nega100** [reply]
Thanks for the respon @Shaobin-Jiang, however I just figure it out how to detect submit button using html-slider plugin. This will work on touchscreen device like smartphone n tablet. All I need to do is to change: 
```
html += '<button id="jspsych-html-slider-response-next" class="jspsych-html-button-response-button jspsych-btn" '+ (trial.require_movement ? "disabled" : "") + '>'+trial.button_label+'</button>';
```
to
```
html += '<input type="submit" id="jspsych-html-slider-response-next" class="jspsych-btn" value="'+trial.button_label+'"></input>';
```
in html-slider-plugins
